packages/core/src/sensors/types.ts

```diff
@@ -81,7 +81,7 @@ export interface Sensor<T extends Object> {
 
 export type Sensors = Sensor<any>[];
 
-export type SensorDescriptor<T> = {
+export type SensorDescriptor<T extends Object> = {
```

Comment:
I use the same boundary is for `Sensor`:
`export interface Sensor<T extends Object> {`